bin/quota

```diff
@@ -377,17 +379,26 @@ function getGroupQuota() {
       #
       # Use project quota for this lfs if configured or default to group quota.
       #
-      if [[ -n "${_lfs:-}" && -n "${quota_pid_increments["${_lfs}"]:-}" ]]; then
-        #echo "DEBUG: _lfs = ${_lfs} and pid increment = ${quota_pid_increments[${_lfs}]}."
-        local _pid
-        local _gid
-        _gid="$(getent group "${_group}" | awk -F ':' '{printf $3}')"
-        _pid=$((${_gid} + ${quota_pid_increments[${_lfs}]}))
-        if [[ "${_lfs}" == 'home' ]]; then
-          _fs="${_fs}/${_group}"
+      #echo "DEBUG: checking for ${_fs_for_report}/.quotacache"
+      if [[ -e "${_fs_for_report}/.quotacache" ]]; then
+        if [[ -r "${_fs_for_report}/.quotacache" ]]; then
+          local _pid
+          if grep -q 'project_id=' "${_fs_for_report}/.quotacache"; then
+            _pid=$(grep 'project_id=' "${_fs_for_report}/.quotacache" | sed 's/project_id=//')
+            #echo "DEBUG: _lfs = ${_lfs} and _pid = ${_pid}."
+            if [[ "${_lfs}" == 'home' ]]; then
+              _fs="${_fs}/${_group}"
+            fi
+            _quota_type='F'
+            _fs_quota="$(lfs quota -q -h -p "${_pid}" "${_fs}" 2> /dev/null | grep -A1 "${_fs}" | grep -v 'is using default' | tr -d '\n')" || reportError "${LINENO}" "${?}"
+          else
+            echo "ERROR: Cannot parse project_id from ${_fs_for_report}/.quotacache"
```

Comment:
```suggestion
            echo "ERROR: Cannot parse project_id from ${_fs_for_report}/.quotacache"
            echo "Perhaps the file was accidentally removed. It will automatically recreated. Try again in an hour.
```

Something like this might help.